frontend/tablet/src/screens/RoomDetailScreen/Header.tsx

```diff
@@ -1,14 +1,25 @@
-import React from 'react'
+import React, { useEffect, useState } from 'react'
 import { View } from 'react-native'
+import moment from 'moment'
 
 import { Typography, Button, Grid } from '../../components'
 
 const Header: React.FC = () => {
+  const [time, setTime] = useState('')
+  const [date, setDate] = useState('')
+
+  useEffect(() => {
+    setInterval(() => {
+      setTime(moment().add(1, 'hours').format('hh:mm'))
+      setDate(moment().format('dddd, MMM DD'))
+    }, 1000)
+  })
+
   return (
     <Grid direction="row" justify="space-between">
       <View>
-        <Typography variant="h3">13:37</Typography>
-        <Typography variant="h4">Thursday, Oct 28</Typography>
+        <Typography variant="h3">{time}</Typography>
+        <Typography variant="h4">{date}</Typography>
```

Comment:
Add the formatting here in render rather than in the state. You can use JS functions to do this and you can check the docs of these functions for extra formatting: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Date/toLocaleString.

![image](https://user-images.githubusercontent.com/43814121/149664987-c786c6d3-bcca-4311-9a92-d39952caf82d.png)